be/src/exec/vectorized/sorting/sort_permute.cpp

```diff
@@ -156,7 +156,7 @@ class ColumnAppendPermutation final : public ColumnVisitorMutableAdapter<ColumnA
 
         auto& offsets = dst->get_offset();
         auto& bytes = dst->get_bytes();
-        size_t old_rows = offsets.size();
+        size_t old_rows = offsets.size() - 1;
         size_t total_rows = offsets.size();
```

Comment:
why don't change `total rows `

Reply:
The semantics of total_rows and old_rows are different， may be misunderstood

Reply:
rename it to `num_offsets`, it's number of offsets, not rows.